### ISSUE TITLE
[Self Request] Magnifier does not show on long press

### ISSUE THREAD

**Issue by antholeole** [flutter/flutter#108823]
Once https://github.com/flutter/flutter/pull/107477 lands, we'll have a magnifier in flutter. Currently, the implementation doesn't show the loupe on long press, but it should. I don't think it's going to take too long but I didn't want to bloat that massive PR, so I'm filing this as a separate issue

**Comment by antholeole** [comment]
unassigned :( got too wrapped up in docs. 

@justinmc @Renzo-Olivares @LongCatIsLooong
